ios/Exponent/Versioned/Core/EXVersionManager.mm

```diff
@@ -327,6 +327,17 @@ - (NSArray *)extraModulesForBridge:(id)bridge
   UMModuleRegistry *moduleRegistry = [moduleRegistryAdapter moduleRegistryForParams:params forExperienceId:experienceId withKernelServices:services];
   NSArray<id<RCTBridgeModule>> *expoModules = [moduleRegistryAdapter extraModulesForModuleRegistry:moduleRegistry];
   [extraModules addObjectsFromArray:expoModules];
+  
+  if (!RCTTurboModuleEnabled()) {
+    [extraModules addObject:[self getModuleInstanceFromClass:[self getModuleClassFromName:"DevSettings"]]];
+    id exceptionsManager = [self getModuleInstanceFromClass:RCTExceptionsManagerCls()];
+    if (exceptionsManager) {
+      [extraModules addObject:exceptionsManager];
+    }
+    [extraModules addObject:[self getModuleInstanceFromClass:[self getModuleClassFromName:"DevMenu"]]];
+    [extraModules addObject:[self getModuleInstanceFromClass:[self getModuleClassFromName:"RedBox"]]];
```

Comment:
I'm nitpicking, but now when I'm thinking of `getModuleInstanceFromClass` I've got a feeling we should rename it to `createModuleInstanceFromClass` because it always creates new instances and doesn't get existing one 🤔

Reply:
This method is a part of `RCTTurboModuleManagerDelegate` protocol, so… no? But if you push this change upstream I'll happily change our method too!